automation/src/test/scala/org/broadinstitute/dsde/test/api/SamApiSpec.scala

```diff
@@ -56,35 +56,27 @@ class SamApiSpec extends AnyFreeSpec with BillingFixtures with Matchers with Sca
   }
 
   "Sam test utilities" - {
-    "should be idempotent for user registration and removal" in {
+    "should be idempotent for removal of user's registration" in {
 
       // use a temp user because they should not be registered.  Remove them after!
 
       val tempUser: Credentials = UserPool.chooseTemp
       val tempAuthToken: AuthToken = tempUser.makeAuthToken()
 
-      //It's possible that some other bad test leaves this user regsistered.
-      //Clean it up if it exists already...
+      // Register user if the user is not registered
       Sam.user.status()(tempAuthToken) match {
         case Some(user) => {
-          logger.info(s"User ${user.userInfo.userEmail} was already registered. Removing before test starts...")
-          removeUser(user.userInfo.userSubjectId)
+          logger.info(s"User ${user.userInfo.userEmail} was already registered.")
+        }
+        case None => {
+          logger.info (s"User ${tempUser.email} does not yet exist! Registering user.")
+          registerAsNewUser(WorkbenchEmail(tempUser.email))(tempAuthToken)
```

Comment:
Can we get away with not calling Orch at all and just calling the Sam `register User` endpoint?   Since these are Sam API tests, I feel like we should just be calling Sam APIs.  Not that I want to keep blowing up the scope on this, but if we're in here fixing it, let's make it as "good" as we can.  If you want CI team to take on making this change, just let us know.

Reply:
Hey @gpolumbo-broad, Anu made a commit that addresses your comment here. Let me know if that looks good to you. https://github.com/broadinstitute/sam/pull/549/commits/4df89b03be49f5ae32a92b3ed0e8dff8bc5a6857

She's out for the remainder of this week, but I'll like to follow up and merge.

In the meantime, I'll rerun the failed Jenkins tests.

Reply:
This PR is the workbench-libs companion - tests on this PR will fail until after the w-libs PR is merged and the dependencies in this PR are updaed https://github.com/broadinstitute/workbench-libs/pull/795